build.sbt

```diff
@@ -10,9 +10,9 @@ ThisBuild / developers := List(
     url("https://github.com/ikhoon")
   )
 )
-ThisBuild / crossScalaVersions := Seq("2.13.8", "2.12.16")
+ThisBuild / crossScalaVersions := Seq("2.13.8")
```

Comment:
`http4s` has dropped support for Scala 2.12 in the `1.x` series, so we have to do the same here. But we exactly can add Scala 3 support (even for `main`). I'm planning to tackle this after a while.